mobile/src/main/java/org/openhab/habdroid/util/Util.java

```diff
@@ -155,6 +161,17 @@ public static void setActivityTheme(@NonNull final Activity activity, String the
             themeRes = R.style.HABDroid_Light;
         }
         activity.setTheme(themeRes);
+
+        if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.LOLLIPOP) {
+            Bitmap icon = BitmapFactory.decodeResource(activity.getResources(), R.mipmap.icon_round);
+            TypedValue typedValue = new TypedValue();
+            Resources.Theme themeActivity = activity.getTheme();
+            themeActivity.resolveAttribute(R.attr.colorPrimary, typedValue, true);
+            @ColorInt int color = typedValue.data;
+            String name = activity.getString(R.string.app_name);
+            ActivityManager.TaskDescription description = new ActivityManager.TaskDescription(name, icon, color);
```

Comment:
Can probably omit the intermediate variable here, can't you?

Reply:
Done!